documentation/docs/release-notes/3.3.0.md

```diff
@@ -131,3 +131,9 @@ When using MySQL Performance Schema as the query source, query examples may be m
 
 #### Workaround
 Use the `slowlog` query source to capture and retain full query examples. For details, see the [About query analytics (QAN)](../use/qan/index.md#limitation-missing-query-examples-in-mysql-performance-schema) topic.
+
+### Incorrect `tls_skip_verify` values in API responses
```

Comment:
This looks good to me. Problem is that it shows up under "Known Issues in QAN" title, which it isn't.